### PR TITLE
Fix context filter has no effect in dashboard

### LINKED ISSUE #26686: Context filter has no effect in dashboard
### Description

Follow #26512

bug

### Gitea Version

latest

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

build

### Database

None

### PR DESCRIPTION
**Pull request by yp05327**
Fix #26686

### CODE REVIEW

**Review by wolfogre — APPROVED** [review]

**Review by wxiaoguang — APPROVED** [review]

File: routers/web/user/home.go [hunk 1]
```diff
@@ -448,8 +448,8 @@ func buildIssueOverview(ctx *context.Context, unitType unit.Type) {
 	// - Team org's owns the repository.
 	// - Team has read permission to repository.
 	repoOpts := &repo_model.SearchRepoOptions{
-		Actor:       ctx.Doer,
-		OwnerID:     ctx.Doer.ID,
+		Actor:       ctxUser,
```
Comment (lunny) [inline routers/web/user/home.go]: Why `Actor` should be `ctxUser`? Maybe it should still be `ctx.Doer`?

Comment (wxiaoguang) [reply routers/web/user/home.go]: Oops, good question.

Comment (yp05327) [reply routers/web/user/home.go]: You are right, Actor should be `ctx.Doer`.  Or doer can see the private repo which can not access.
Copied from Milestones render function. We should also fix it. 😕

Comment (yp05327) [reply routers/web/user/home.go]: Done in https://github.com/go-gitea/gitea/pull/26695/commits/f5dd6b5072551c6001e26f7169ee756ad6de4cfc

**Review by delvh — APPROVED** [review]

**Review by wxiaoguang — APPROVED** [review]

**Review by wolfogre — APPROVED** [review]

**Comment by puni9869** [comment]
lets. roll this 🚀

**Pull request merged**